src/nested_pandas/series/ext_array.py

```diff
@@ -648,6 +648,12 @@ def num_chunks(self) -> int:
         """Number of chunks in underlying pyarrow.ChunkedArray"""
         return self._chunked_array.num_chunks
 
+    @property
+    def list_index(self) -> np.ndarray:
+        """Keys mapping values to lists"""
```

Comment:
This docstring is short enough to be a little puzzling; there's a lot going on in a few statements here.

Reply:
Name is also not great, I'd appreciate a help with renaming

Reply:
The idea was this index mapping "flat value" to "list value"